internal/pkg/manifest/workload.go

```diff
@@ -187,24 +181,16 @@ type Logging struct {
 	ConfigFile     *string           `yaml:"configFilePath"`
 }
 
-func (lc *Logging) logConfigOpts() *template.LogConfigOpts {
-	return &template.LogConfigOpts{
-		Image:          lc.image(),
-		ConfigFile:     lc.ConfigFile,
-		EnableMetadata: lc.enableMetadata(),
-		Destination:    lc.Destination,
-		SecretOptions:  lc.SecretOptions,
-	}
-}
-
-func (lc *Logging) image() *string {
+// LogImage returns the default Fluent Bit image if not otherwise configured.
+func (lc *Logging) LogImage() *string {
```

Comment:
now that `LogImage` and `GetEnableMetadata` are public can we add unit tests for them?